yoti-sdk-api/src/main/java/com/yoti/api/client/Anchor.java

```diff
@@ -10,23 +10,25 @@
 public interface Anchor {
 
     /**
-     * The type of Anchor - most likely either SOURCE or VERIFIER, but it's possible that new Anchor types will be added in future
-     *
+     * The type of Anchor - most likely "SOURCE" or "VERIFIER" but it's possible that new Anchor types will be added in future
      * @return String, naming the type of the anchor
      */
     String getType();
 
     /**
-     * For SOURCE Anchors, describes how the attribute value was obtained, for example this might be 'OCR' or 'NFC'
-     * For VERIFIER Anchors, describes the verification method used for the attribute
+     * <pre>
+     * For SOURCE Anchors, describes how the attribute value was obtained, will be one of: ['USER_PROVIDED', 'PASSPORT', 'DRIVING_LICENSE', 'PASSCARD', 'NATIONAL_ID']
```

Comment:
Sorry Alex, I missed this the first time. Could we change the phrasing to say that the options listed are _among_ the possible options (rather than `will be one of`), so that we're not saying that the ones listed are the only options. We also can include `YOTI_AI` as an option now. (see .[NET](https://github.com/getyoti/yoti-dotnet-sdk/pull/105/files))